ext/ext.md

```diff
@@ -2,7 +2,7 @@
 
 ## Summary
 
-This base extension is used to discover and return an array of extensions.
+This base extension namespace is used to discover and return an array of extensions.
```

Comment:
This namespace concept needs some words of definition what it is namespacing, right?

Reply:
@vbatts  My understanding is it is namespacing the extensions supported by distribution. The base extension namespace includes discover component that lists all the available extensions.

Reply:
@vbatts @mnltejaswini can we get the baseline in and reword since we agreed to get this in some form and iterate with updates. 
I think we need this to make progress on the implementation PR and ORAS path updates as well.